commcare_export/excel_query.py

```diff
@@ -180,16 +180,18 @@ def compile_source(worksheet):
         return FlatMap(source=api_query,
                        body=Reference(str(data_source_jsonpath)))
 
-def compile_sheet(worksheet, mappings=None, missing_value=None):
+def compile_sheet(worksheet, mappings=None, missing_value=None, ignore_empty=True):
     mappings = mappings or {}
     source_expr = compile_source(worksheet)
 
     output_table_name = worksheet.title
-    output_headings = get_column_by_name(worksheet, 'field') # It is unfortunate that this is duplicated here and in `compile_fields`
+    output_headings = get_column_by_name(worksheet, 'field')  # It is unfortunate that this is duplicated here and in `compile_fields`
     output_fields = compile_fields(worksheet, mappings=mappings)
 
     if not output_fields:
-        headings = headings = []
+        if ignore_empty:
```

Comment:
I'm having trouble understanding this variable name- to me ignore_empty implies that not having output fields _wouldn't_ raise an exception?